modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -133,26 +134,35 @@ public AbstractJavaCodegen() {
-                        "localVarPath", "localVarQueryParams", "localVarCollectionQueryParams",
-                        "localVarHeaderParams", "localVarCookieParams", "localVarFormParams", "localVarPostBody",
-                        "localVarAccepts", "localVarAccept", "localVarContentTypes",
-                        "localVarContentType", "localVarAuthNames", "localReturnType",
-                        "ApiClient", "ApiException", "ApiResponse", "Configuration", "StringUtil",
-
-                        // language reserved words
-                        "abstract", "continue", "for", "new", "switch", "assert",
-                        "default", "if", "package", "synchronized", "boolean", "do", "goto", "private",
-                        "this", "break", "double", "implements", "protected", "throw", "byte", "else",
-                        "import", "public", "throws", "case", "enum", "instanceof", "return", "transient",
-                        "catch", "extends", "int", "short", "try", "char", "final", "interface", "static",
-                        "void", "class", "finally", "long", "strictfp", "volatile", "const", "float",
-                        "native", "super", "while", "null")
-        );
+        List<String> words = new ArrayList<>(Arrays.asList(
+                // special words
+                "object",
+                // used as internal variables, can collide with parameter names
+                "localVarPath", "localVarQueryParams", "localVarCollectionQueryParams",
+                "localVarHeaderParams", "localVarCookieParams", "localVarFormParams", "localVarPostBody",
+                "localVarAccepts", "localVarAccept", "localVarContentTypes",
+                "localVarContentType", "localVarAuthNames", "localReturnType",
+                "ApiClient", "ApiException", "ApiResponse", "Configuration", "StringUtil",
+
+                // language reserved words
+                "abstract", "continue", "for", "new", "switch", "assert",
+                "default", "if", "package", "synchronized", "boolean", "do", "goto", "private",
+                "this", "break", "double", "implements", "protected", "throw", "byte", "else",
+                "import", "public", "throws", "case", "enum", "instanceof", "return", "transient",
+                "catch", "extends", "int", "short", "try", "char", "final", "interface", "static",
+                "void", "class", "finally", "long", "strictfp", "volatile", "const", "float",
+                "native", "super", "while", "null"
+        ));
+        try {
+            ClassPath.from(ClassLoader.getSystemClassLoader())
+                    .getTopLevelClasses("java.lang").stream()
+                    .map(ClassPath.ClassInfo::getSimpleName)
+                    .filter(StringUtils::isNotBlank)
+                    .forEach(words::add);
+        } catch (IOException e) {
+            throw new UncheckedIOException("Couldn't load classes from java.lang", e);
+        }
```

Comment:
The list of classes will depend on the version of the JVM being used in the generator.
As an alternative, we could list the classes explicitly (although this would be a very long list).

Reply:
The list is empty with JDK11 and 14. I've tried to upgrade guava to `30.0-jre` without a better result.

I don't know if it is still possible to list java.lang classes. Maybe we must continue with the hand-made list.

Reply:
@Zomzog Thanks for looking into this!

I could reproduce the issue with Java 11 and 15. 😢
Subsequently, I've opted to go with a static list of classes in `java.lang`.